src/scripts.js

```diff
@@ -1 +1,26 @@
-console.log("Hello World");
\ No newline at end of file
+let userRepo = new UserRepository(userData);
+
+const userName = document.querySelector('.user-name');
+const userAddress = document.querySelector('.user-address');
+const userEmail = document.querySelector('.user-email');
+const userStride = document.querySelector('.user-stride-length');
+const userStepGoal = document.querySelector('.user-step-goal');
+const userGoalAverage = document.querySelector('.step-goal-average');
+const friendsList = document.querySelector('.friends-list');
+const stepComparison = document.querySelector('.step-goal-comparison')
+
+function populateUserInfo(id) {
+  let user = new User(userRepo.getUserData(id))
+  userName.innerHTML = user.name;
```

Comment:
Why are we using innerHTML instead of innerText if we are only changing the text every time?

Reply:
Switched